apps/remix-ide/src/app.js

```diff
@@ -388,11 +400,22 @@ class AppComponent {
       this.engine.register([xterm])
       const ripgrep = new ripgrepPlugin()
       this.engine.register([ripgrep])
+      const appUpdater = new appUpdaterPlugin()
+      this.engine.register([appUpdater])
+      const remixAIDesktop = new remixAIDesktopPlugin()
+      this.engine.register([remixAIDesktop])
     }
+    
+    const remixAI = new RemixAIPlugin(isElectron())
+    this.engine.register([remixAI])
```

Comment:
@yann300 activate this plugin on startup, since it is not view anymore and requires user activate